go/adbc/driver/snowflake/statement.go

```diff
@@ -253,7 +331,7 @@ func toSnowflakeType(dt arrow.DataType) string {
 		if ts.TimeZone == "" {
 			return fmt.Sprintf("timestamp_ntz(%d)", prec)
 		}
-		return fmt.Sprintf("timestamp_tz(%d)", prec)
+		return fmt.Sprintf("timestamp_ltz(%d)", prec)
```

Comment:
We're changing it from `tz` to `ltz` here. The reason for the change is that the Snowflake [TIMESTAMP_LTZ](https://docs.snowflake.com/en/sql-reference/data-types-datetime#timestamp-ltz-timestamp-ntz-timestamp-tz) type seems to be much more closely aligned with Arrow Timestamp semantics. Both Arrow and the `ltz` variant store time since UTC epoch, along with timezone metadata used primarily for presentation in applications. The `tz` variant stores a fixed offset which is something that Arrow doesn't represent natively. So we can read either variant and handle offsets, etc but it's preferred to write only the `ltz` variant when starting from an Arrow type.

Reply:
Gotcha, that makes more sense.